random-ballerina/build.gradle

```
@@ -210,6 +213,8 @@ task ballerinaPublish {
                     commandLine 'sh', '-c', "$distributionBinPath/bal push ${packageName}"
                 }
             }
+        } else {
+            println("Central Access Token is not present")
```

It's better to have it this way. We'll update this in the other repos.

```suggestion
            throw new InvalidUserDataException("Central Access Token is not present")
```